jobserver/views/admin.py

```diff
@@ -0,0 +1,64 @@
+from django import forms
+from django.contrib import messages
+from django.db import transaction
+from django.shortcuts import redirect
+from django.urls import reverse_lazy
+from django.utils.decorators import method_decorator
+from django.views.generic import FormView
+
+from ..authorization.decorators import require_superuser
+from ..models import Backend, BackendMembership, Org, OrgMembership, User
+
+
+class ApprovalForm(forms.Form):
+    def __init__(self, backends, orgs, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # bulid the backends and orgs fields based on the values passed in
+        self.fields["backends"] = forms.ModelMultipleChoiceField(queryset=backends)
+        self.fields["orgs"] = forms.ModelMultipleChoiceField(queryset=orgs)
+
+
+@method_decorator(require_superuser, name="dispatch")
+class ApproveUsers(FormView):
+    form_class = ApprovalForm
+    success_url = reverse_lazy("admin:jobserver_user_changelist")
+    template_name = "approve_users.html"
+
+    def dispatch(self, request, *args, **kwargs):
+        pks = self.request.GET.getlist("user")
+
+        if not pks:
+            messages.error(request, "One or more users must be selected")
+            return redirect(self.success_url)
```

Comment:
Should we define `failure_url` too?

Reply:
It's the same URL and I'm copying the GCBV paradigm for the name, but I see your point… I think I'll leave it for now and address if I end up adding anything to this view.

Reply:
Totally fair enough!